projects/components/src/summary-list/summary-list.component.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { PrimitiveValue } from '@hypertrace/common';
+import { startCase } from 'lodash-es';
+import { SummaryItem } from './summary-list-api';
+
+@Component({
+  selector: 'ht-summary-list',
```

Comment:
I don't love the name (when do I ever?). We already have a list-view, card-list, summary-values, table - so from the names, it's not clear how this one is different and when to use which.

Reply:
Yeah, I was scanning through our components and thinking the same thing. Open to suggestions. :-)

Reply:
Ah, of course - my own words turned against me. In that case, carry on.